pkg/net/libp2p/key.go

```diff
@@ -27,13 +27,12 @@ func operatorPrivateKeyToNetworkKeyPair(operatorPrivateKey *operator.PrivateKey)
 		return nil, nil, fmt.Errorf("libp2p supports only secp256k1 operator keys")
 	}
 
-	// Libp2p keys are actually btcec keys under the hood.
-	btcecPrivateKey, btcecPublicKey := btcec.PrivKeyFromBytes(
-		DefaultCurve, operatorPrivateKey.D.Bytes(),
+	btcecPrivateKey, btcecPublickKey := btcec.PrivKeyFromBytes(
```

Comment:
There is a typo:
```suggestion
	btcecPrivateKey, btcecPublicKey := btcec.PrivKeyFromBytes(
```

Reply:
Fixed in 8f7e7044e4e1a9a9208102f103ee3243369ffb33.